app/controllers/v0/education_career_counseling_claims_controller.rb

```diff
@@ -5,7 +5,7 @@ class EducationCareerCounselingClaimsController < ClaimsBaseController
     def create
       load_user
       claim = SavedClaim::EducationCareerCounselingClaim.new(form: filtered_params[:form])
-      claim.add_claimant_info(current_user) if current_user
+      claim.add_claimant_info(current_user) if current_user&.loa3?
```

Comment:
can we get some tests for creating a claim that's not a loa3 user? (or one that does expect loa3 user?  interesting the existing tests didn't break)

Reply:
For sure. Tests added no problem.

> interesting the existing tests didn't break
Might be statin' the obvious but I was loading an LOA3 user in the tests. So essentially, the code was doing the same thing it was before we added these changes.